hcl/printer/printer.go

```diff
@@ -60,5 +60,8 @@ func Format(src []byte) ([]byte, error) {
 		return nil, err
 	}
 
+	// Add trailing newline to result
+	buf.WriteString("\n")
```

Comment:
It's already broken on windows - we opt out of the tests there. I did a quick google search on the topic, and `\n` is what Println does, and windows deals with it. Hence, this.